Formula/clisp.rb

```diff
@@ -1,19 +1,15 @@
 class Clisp < Formula
   desc "GNU CLISP, a Common Lisp implementation"
   homepage "https://clisp.sourceforge.io/"
-  revision 2
+  license "GPL-2.0-or-later"
 
   stable do
-    url "https://ftp.gnu.org/gnu/clisp/release/2.49/clisp-2.49.tar.bz2"
-    mirror "https://ftpmirror.gnu.org/clisp/release/2.49/clisp-2.49.tar.bz2"
-    sha256 "8132ff353afaa70e6b19367a25ae3d5a43627279c25647c220641fed00f8e890"
+    url "https://alpha.gnu.org/gnu/clisp/clisp-2.49.92.tar.bz2"
+    sha256 "bd443a94aa9b02da4c4abbcecfc04ffff1919c0a8b0e7e35649b86198cd6bb89"
 
+    # Fix build on ARM.
+    # Remove once https://gitlab.com/gnu-clisp/clisp/-/commit/39b68a14d9a1fcde8a357c088c7317b19ff598ad is released.
     patch :DATA
```

Comment:
How is the patch related to the commit in your comment?

Reply:
A change similar to that of the end of file patch was made to Gnulib: https://git.savannah.gnu.org/gitweb/?p=gnulib.git;a=commit;h=00e688fc22c7bfb0bba2bd8a7b2a7d22d21d31ef Then the Gnulib bundled with clisp got an update (that's why the clisp commit is massive).